data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -183,6 +183,21 @@
       <arg type="o" name="request_handle" direction="out"/>
     </method>
 
+    <!--
+        ConfigureShortcuts:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object representing the session
+        @options: Vardict with optional further information
```

Comment:
This lacks documentation on what is supported to pass here for the options dict

Reply:
This not done anywhere else where there is an option dict but no option yet.